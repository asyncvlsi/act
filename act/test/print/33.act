defproc test()
{
  int a, b, c;

  chp {
    a := b << c
  }
}

test t;
