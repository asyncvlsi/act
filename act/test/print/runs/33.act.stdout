defproc test ();

defproc test ()
{

/* instances */
int<32> b;
int<32> a;
int<32> c;

/* connections */
chp {
a:=(c>>0x6)!=0x0 ? int(0x0,32) : int(b<<int(c,6),32)
}
}


/* instances */
test t;

/* connections */
